ShopNow/UserApp/views.py

```diff
@@ -41,40 +54,59 @@ def registerUser(request):
                 serialized.save()
                 # print(f"""Username: {serialized.data.get('username')}, Email: {serialized.data.get('useremail')},
                 #     Password: {serialized.data.get('password')}, Confirm Password: {serialized.data.get('confirm_password')}""")
-                return Response(serialized.data)
+                return Response(serialized.data,status=status.HTTP_201_CREATED)
             else:
-                return Response("your data is invalid")
+                return Response(f"your data is invalid,{serialized.errors}",status=status.HTTP_400_BAD_REQUEST)
         else:
```

Comment:
We do not need to check request.data here as serializer will handle this. To check the validity of serailizer is enough here. so we can remove this if and else.